### PR TITLE
Update SDK js tutorials

### PR DESCRIPTION
**Pull request by FrankEssenberger**
Added a comment about passwords as indicated here: https://github.com/SAPDocuments/Issues/issues/526
Also added not actively maintained anymore annotation.

### CODE REVIEW

File: tutorials/cloudsdk-js-vdm-getall/cloudsdk-js-vdm-getall.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ primary_tag: products>sap-cloud-sdk
  - Basic knowledge of OData is recommended, but not required.
 
 ## Details
+
+> ### DEPRECATED
+> This tutorial on the SAP Cloud SDK for JavaScript is not maintained anymore and is probably outdated.
+> Up-to-date documentation is published on the [documentation portal](https://sap.github.io/cloud-sdk/)  of the SAP Cloud SDK.
```
Comment (artemkovalyov) [inline tutorials/cloudsdk-js-vdm-getall/cloudsdk-js-vdm-getall.md:19]:
```suggestion
> ### We migrate tutorials to our [documentation](https://sap.github.io/cloud-sdk/docs/js/getting-started)
> This tutorial is not actively maintained and might be partially outdated.
> Always up-to-date documentation is published on our [documentation portal](https://sap.github.io/cloud-sdk/docs/js/getting-started).
> We will provide a link to the updated version of this tutorial as soon as we release it.
```

Comment (artemkovalyov) [reply tutorials/cloudsdk-js-vdm-getall/cloudsdk-js-vdm-getall.md:19]: I'd avoid using `DEPRECATED`. Many things a quite contextual, see my suggestion.

**Review by artemkovalyov — CHANGES_REQUESTED** [review]
I left my version in the suggestion. Let me know what you think.

**Review by artemkovalyov — APPROVED** [review]
Thanks. Let's spread it around other tutorials.

**Comment by FrankEssenberger** [comment]
@sacnl I wanted to add you as reviewer, but I could not assign more reviewer.

**Pull request merged**